### PR TITLE
Add TinyMS 0.1.0 release note

### PR DESCRIPTION
**Pull request by leonwanghui**
**What type of PR is this?**

`/kind documentation`</br>

**What does this PR do / why do we need it**:

Add the TinyMS `0.1.0` release note.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewers**:

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
+* Support install TinyMS binary in `Linux Ubuntu 18.04` and `Window 10` environment, also provide TinyMS docker image to users. [#2](https://github.com/tinyms-ai/tinyms/pull/2) [#45](https://github.com/tinyms-ai/tinyms/pull/45)
+* Enable document auto-generation using Sphinx. [#35](https://github.com/tinyms-ai/tinyms/pull/35)
+* Provide several end to end model development and deployment [tutorials](https://tinyms.readthedocs.io/en/latest/quickstart/quickstart_in_one_minute.html) for machine learning beginners. [#11](https://github.com/tinyms-ai/tinyms/pull/11) [#24](https://github.com/tinyms-ai/tinyms/pull/24) [#26](https://github.com/tinyms-ai/tinyms/pull/26) [#34](https://github.com/tinyms-ai/tinyms/pull/34)
+* Set up the initial CI pipeline (including cla-assistant, GitHub Actions, readthedocs) for TinyMS project. [#1](https://github.com/tinyms-ai/tinyms/pull/1) [#49](https://github.com/tinyms-ai/tinyms/pull/49) [#50](https://github.com/tinyms-ai/tinyms/pull/50)
+
 #### NewModels
```
Comment (hannibalhuang) [inline RELEASE.md:15]: s/NewModels/Model Park

**Review by hannibalhuang — CHANGES_REQUESTED** [review]
small fix

File: RELEASE.md (hunk 1 above)
Comment (leonwanghui) [reply RELEASE.md:15]: Got it

**Comment by leonwanghui** [comment]
@hannibalhuang Please reivew it again.

**Review by hannibalhuang — APPROVED** [review]
LGTM

**Pull request merged**